ringersong/src/main/java/com/RingerSong/free/service/RingerCallScreeningService.kt

```
@@ -0,0 +1,83 @@
+                        .build()
+
+                    respondToCall(callDetails, response)
+
+                    // Start our custom playback
+                    startPlaybackService(phoneNumber)
+
+                } else {
+                    Log.d(TAG, "RingerSong disabled. Allowing call.")
+                    respondToCall(callDetails, CallResponse.Builder().build())
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)